client/src/holdings/HoldingsList.tsx

```diff
@@ -62,9 +62,10 @@ export default function HoldingsList({user}: HoldingsListProps) {
     setOpen([...open]);
   };
 
-  const addHolding = (holding: Holding) => {
-    holdings.push(holding);
-    setHoldings([...holdings]);
+  const addHolding = (holding: Holding) => { 
+    holdings.push(holding); 
+    holdings.sort((a, b) => (a.name.toLowerCase() > b.name.toLowerCase()) ? 1 : -1);
+    setHoldings([...holdings]); 
```

Comment:
As discussed need to remove rebase your new work off head to remove this previous stuff.

Also your commit title should be

[Client] Add instrument symbol to the 'add trade dialog' header (fix #32)

Reply:
No worries - should be done now

Reply:
Looks like the old commits are still here? Might be easier to just make a fresh branch off master.